OneCommunityGlobal/HighestGoodNetworkApp
# Raul improve owner message feature

# Description
(DASHBOARD COMPONENT - PRIORITY LOW) - New Requests

> (PRIORITY LOW) Jae: Add to top of app a section that the “Owner” class (only) can edit to post a message to all users of the app like, “Happy Holidays”. (WIP - Raul)
> I’d like the UI for posting the message to have the ability for me to set an expiration date so it’ll automatically return to the logo on that date
> **New Requests:** 
> I. Change “or add a picture” to “Or add a picture (max size ## x ## pixels and XX KB).
> II. Please add a “Clear All” button that will remove whatever has been saved there and return it to blank. (A big red X)

## Mainly changes explained:
A new feature was added to the owner message component. Now, we have a standard message that is displayed when no message is submitted. Also, it was installed the option of changing this standard message. New icons were put there as well. The clear/remove icon just displays when we have a common message.

## How to test:
1. this PR is related to backend [PR #288](https://github.com/OneCommunityGlobal/HGNRest/pull/288). You have to use both to test this feature.
2. git checkout raul_improve_owner_message_feature
3. npm install
4. npm run start:local
5. log in as **OWNER**
6. A pen should be displayed for you:
![image](https://user-images.githubusercontent.com/29555732/227401999-b94239e9-e77b-493b-847c-989b67b4ec88.png)

7. click and try to update the standard message. Also, create common messages to test. For image messages, try using these - the resolutions were made for this resource:
![image](https://user-images.githubusercontent.com/29555732/227402343-d04713fe-7f6d-437d-a6ec-c8e9996180f9.png)
![Logo](https://user-images.githubusercontent.com/29555732/227402344-99f8af5e-fe39-43a7-ad56-ffd9eb2806f1.png)

8. The pen should only be displayed for owner accounts. The messages have to be displayed for everyone.
9. This resource is made only for computers (width higher than 1400px).
